src/Objs/Homepage/HomepageLayoutComponent.tsx

```diff
@@ -1,7 +1,37 @@
-import { provideLayoutComponent } from 'scrivito'
-import { Homepage } from './HomepageObjClass'
+import {
+  connect,
+  CurrentPage,
+  currentPage,
+  Obj,
+  provideLayoutComponent,
+} from 'scrivito'
+import { Homepage, HomepageInstance } from './HomepageObjClass'
 import { DefaultPageLayoutComponent } from '../defaultPageLayoutComponent'
+import { Loading } from '../../Components/Loading'
 
-provideLayoutComponent(Homepage, ({ page }) => {
-  return <DefaultPageLayoutComponent page={page} />
-})
+provideLayoutComponent(Homepage, ({ page }) => <HomepageLayout page={page} />)
+
+// TODO: Remove workaround for issue #11635 once it is fixed.
+const HomepageLayout = connect(
+  function HomepageLayout({ page }: { page: HomepageInstance }) {
+    if (ignoreHomepageLayout()) return <CurrentPage />
+
+    return <DefaultPageLayoutComponent page={page} />
+  },
+  { loading: Loading },
+)
+
+function ignoreHomepageLayout(): boolean {
+  const firstSubpagePath = currentPage()
+    ?.path()
+    ?.split('/')
+    .slice(0, 2)
+    .join('/')
+  if (!firstSubpagePath) return false
+  if (firstSubpagePath === '/') return false
+
+  const firstSubpage = Obj.getByPath(firstSubpagePath)
+  if (!firstSubpage) return false
```

Comment:
* [ ] FYI Very hard to read. Are we trying to save on requests for `ancestors`? In the end it could lead to an extra request for the path while we already have ancestors (for breadcrumbs). I'd suggest to switch to ancestors-based logic.

Reply:
I extracted parts of this into a dedicated function. With this I hope the it's easier to understand.